app.coffee

```diff
@@ -24,6 +24,7 @@ app.post '/user/:user_id', controller.addNotification
 app.get '/user/:user_id', controller.getUserNotifications
 app.del '/user/:user_id/notification/:notification_id', controller.removeNotificationId
 app.del '/user/:user_id', controller.removeNotificationKey
+app.del '/notification/key/:key', controller.removeNotificationByKeyOnly
```

Comment:
perhaps not, the odds of `notifications` ever managing another object aside from Notification are slim :)